scripts/preprocess.py

```diff
@@ -17,6 +17,17 @@ def process_precip_2018():
                          regrid=None,
                          parallel=False)
 
+def process_vhi_2018():
```

Comment:
this is a test kind of script right? shall I move some of mine (once cleaned) from the [`esowc_notes/example_scripts` repo](https://github.com/tommylees112/esowc_notes/tree/master/example_scripts) to here?

Reply:
yea, that would be awesome